cypress/e2e/utils/room-utils.ts

```diff
@@ -0,0 +1,51 @@
+import Chainable = Cypress.Chainable;
+
+export default class RoomUtils {
```

Comment:
The convention to do this seems to be to add a file in /cypress/support and create `cy.` functions. 
This works too, I guess. 
It's just hard to know where things are if they are in two different places... 🤷‍♀️